### ISSUE TITLE
Use case: Use semantic data for accessibility

### ISSUE THREAD

**Issue by opyh** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#233]
As discussed in https://github.com/Maps4HTML/MapML/issues/130 and in https://discourse.wicg.io/t/physical-accessibility-data-in-maps/4852/5, here is a use case for map + screen reader accessibility:

- Have a map with 15 places of interest
- The places of interest are `feature`s and have very basic semantic RDFa markup
- The features are typed as subtypes of schema.org `Place` (e.g. `Hotel`, `FoodEstablishment`)
- The prototype adds an `aria-label` on the map element that summarizes the visible information
- An exemplary `aria-label` would read: `"a map displaying two hotels, one shopping center, and thirteen more places"`

MapML/RDFa feature code example:

```xml
<feature vocab="https://schema.org" typeof="Hotel">
     <properties>
        <span property="name">An awesome hotel 🏨</span>
     </properties>
     <geometry> ... </geometry>
</feature>
```

We could later add use cases related to physical accessibility, e.g. letting the user agent change the visual appearance of displayed features depending on their physical accessibility features, as outlined in the presentation.

**Comment by prushforth** [comment]
I'd like to get comments from others about this use case, but my initial impression is that it would fit as a "[content author use case](https://maps4html.org/HTML-Map-Element-UseCases-Requirements/#content-author-use-cases)". 

Aside: although the distinction can be blurry, I've often though of "map author" as a role that could (sometimes) be distinct from "html author", especially if the content is separate from the page into which it's being included, as foreseen by the `<layer src="..."></layer>` construct.

Maybe there's an assumption here that a

```html
<feature vocab="https://schema.org" typeof="Hotel">
     <properties>
        <span property="name">An awesome hotel 🏨</span>
     </properties>
     <geometry> ... </geometry>
</feature>
```

element is a requirement, so please advise on that topic as well, and how it may fit into the abstract use cases presented by our document.

**Comment by Malvoz** [comment]
@opyh Just to clarify, this isn't about _requiring_ user agents to display accessibility features based on semantic data is it? But rather to "simply" support the use case of marking up features using semantic data?

I think we can narrow this issue down to the use case of supporting semantic data (RDFa, [Microdata](https://html.spec.whatwg.org/multipage/microdata.html), [Personalization attributes](https://github.com/w3c/personalization-semantics/wiki/Explainer-for-Personalization-Semantics), etc.) _in general_, as that'd cover the described use case (semantic data for accessibility).

> - The prototype adds an `aria-label` on the map element that summarizes the visible information
> - An exemplary `aria-label` would read: `"a map displaying two hotels, one shopping center, and thirteen more places"`

I may disagree with the above, but if that is a suggestion for the prototype, please raise an issue in https://github.com/Maps4HTML/Web-Map-Custom-Element (see related PR: https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/151) and we can discuss!

> Maybe there needs to be a use case like "Need a semantic `<map>` element to be the container for maps"

Or rather, we can add a note about potentially requiring an implicit ARIA role in the Conclusion for [Capability: Embed an interactive map viewer, using HTML markup](https://maps4html.org/HTML-Map-Element-UseCases-Requirements/#capability-declarative-map) (and label it "[Accessibility: potential improvement](https://maps4html.org/HTML-Map-Element-UseCases-Requirements/#accessibility-potential)").

**Comment by opyh** [comment]
Yes, the idea was to add RDFa support in general, because it would allow for a lot of interesting things to happen. That it makes a lot of map accessibility use cases possible (e.g. browser plugins that change maps on the fly) is a subset.

I know that `aria-label` etc. isn't part of the spec yet, either (it should be though).

Should we split this into two use cases then? One for RDFa, one for direct accessibility label support?

**Comment by Malvoz** [comment]
> From what I see, `aria-label` etc. isn't part of the spec yet

The MapML spec does allow global `aria-*` attributes on [the `<map>` element.](https://maps4html.org/MapML/spec/#the-map-element)
